src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -244,6 +244,8 @@ class SeekBar extends Slider {
     super.handleMouseUp(event);
 
     this.player_.scrubbing(false);
+    // forces timeui to not use cache
+    this.player_.tech_.trigger('timeupdate');
```

Comment:
is it necessary to trigger on the tech or can we trigger it on the player itself?

Also, we should trigger this event like we do here: https://github.com/videojs/video.js/blob/master/src/js/tech/tech.js#L304-L310
and we should copy over and update the jsdoc as well like there.

Reply:
It is necessary to trigger on the tech, I had tried `this.player_.trigger`, it didn't work. I will update the trigger event.

Reply:
cool, thanks @shahlabs!

Reply:
Retested with this.player_.trigger and it worked, so updating the trigger on player @gkatsev